Demo/Demo/SwiftUIComponents/CurrentState.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+enum CurrentState: Equatable {
```

Comment:
Ideally we can use this across the other demo features as well vs the individual states per feature that exist today